libsquoosh/src/image_data.ts

```diff
@@ -1,9 +1,13 @@
 export default class ImageData {
-  readonly data: Uint8ClampedArray;
+  readonly data: Uint8ClampedArray | Uint8Array;
```

Comment:
I saw your justification, but I don’t think we should work around the problem this way. I’d rather add an explicit copy (or even just a cast) where needed. I worry about `ImageData` not being fully compatible with the web’s defintion.

Reply:
I think we can just change `resizer.resize` to return `Uint8ClampedArray` for consistency - it just needs wrapping of the return type into [`Clamped`](https://docs.rs/wasm-bindgen/0.2.74/wasm_bindgen/struct.Clamped.html).

Reply:
(although I guess this might break other callsites where we expect `Uint8Array`?)

Reply:
I don’t think we expect that anywhere. Or rather, we shouldn’t.

Reply:
Ah, I was worried about inputs to codecs, but those are `BufferSource` which covers all typed arrays including `Uint8ClampedArray`, so yeah, we should be fine.

Reply:
Makes sense! I updated `resizer.resize` to return `Uint8ClampedArray` in https://github.com/GoogleChromeLabs/squoosh/pull/1044/commits/5707eeff411f1976492968f319692219e9d76fe3

Reply:
Nice!